ws-security-common/pom.xml

```diff
@@ -93,9 +92,13 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.sun.mail</groupId>
-            <artifactId>jakarta.mail</artifactId>
-            <version>${jakarta.mail.version}</version>
+            <groupId>jakarta.mail</groupId>
+            <artifactId>jakarta.mail-api</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.angus</groupId>
+            <artifactId>angus-mail</artifactId>
             <scope>compile</scope>
```

Comment:
This dependency is not actually needed in the tests. Did you check if it's required by the CXF ws-security system tests? Otherwise we could just remove it.

Reply:
It's not required by CXF and we can remove it.  I already changed this PR. Please review.